multiqc/templates/default/content.html

```diff
@@ -13,7 +13,12 @@
     {% for s in m.sections %}
       {% if loop.first %}
       <div class="mqc-module-section-first">
-        <h2 id="{{ m.anchor }}">{{ m.name }}</h2>
+        <div class="mqc-module-title-section">
+          <div><h2 class="mqc-module-title" id="{{ m.anchor }}">{{ m.name }}</h2></div>
+          {% if m.versions %}
+            <div><em>Version{% if m.versions|length > 1 %}s{% endif %}:</em> <code>{{m.versions|join("</code>, <code>")}}</code></div>
```

Comment:
Prefer not to use `<em>` - use CSS instead (with `<span>` + class if needed).

Reply:
```suggestion
            <div class="mqc-module-versions"><span>Version{% if m.versions|length > 1 %}s{% endif %}:</span> <code>{{m.versions|join("</code>, <code>")}}</code></div>
```

Reply:
Actually, just use bootstrap classes to drop the text colour a bit, and I'm being hypocritical and using `<em>` already elsewhere, so meh - probably fine.. 😅 

```suggestion
            <div class="text-muted"><em>Version{% if m.versions|length > 1 %}s{% endif %}:</em> <code>{{m.versions|join("</code>, <code>")}}</code></div>
```